### ISSUE TITLE
Patch to fix extension in Chrome

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Imbubblezz/google-opt-out-plugin#21]
```
Replaced all broken `chrome.experimental.cookie` references to the modern 
`chrome.cookie`.  (This update should also be pushed to the Chrome Web Store, 
as the extension is current broken.)
```

Original issue reported on code.google.com by `apsill...@gmail.com` on 3 May 2012 at 12:55

Attachments:
- [background.js.patch](https://storage.googleapis.com/google-code-attachments/google-opt-out-plugin/issue-21/comment-0/background.js.patch)

**Comment by GoogleCodeExporter** [comment]
```
should be fixed as of version 1.2
```

Original comment by `eisinger@google.com` on 15 Aug 2012 at 8:23
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
,, thank you very much patch for developing a fix for the Chrome browser on the 
ad on the plugin to opt out of the shower malicious ads you're a lifesaver fuck 
it ever do anything to help you you have my email thank you so much
```

Original comment by `JamesPie...@gmail.com` on 26 Mar 2014 at 2:42
